COLLECTOR_VERSION

```diff
@@ -1 +1 @@
-3.19.x-70-gf3a66fa3a5
+3.19.x-99-g0de15fb7b4
```

Comment:
This is so that a version of collector capable of receiving the configuration is used. This will be reverted before merging.